.gitignore

```diff
@@ -79,3 +79,6 @@ docs/.static/api/
 db_data
 es_data
 gem_cache
+
+# Auto-generated Storybook stories
+app/javascript/generated_stories/
```

Comment:
This folder is a build artifact so it should never be committed. It's in the `app/jaavascript` folder so that the stories get picked up by Storybook.